### PR TITLE
Angular tests subgenerator and test added

### PR DESCRIPTION
**Pull request by UndeadBaneGitHub**

### CODE REVIEW

File: angular-tests/templates/_.client.controller.tests.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+(function() {
+	// <%= humanizedControllerName %> Controller Spec
+	describe('<%= humanizedControllerName %> Controller Tests', function() {
```
Comment (codydaig) [inline angular-tests/templates/_.client.controller.tests.js]: Spacing to 2 spaces.

**Comment by codydaig** [comment]
Spacing to 2 spaces. 

Otherwise, LGTM.

**Comment by codydaig** [comment]
@ilanbiala LGTY?

**Comment by ilanbiala** [comment]
LGTM.

**Pull request merged**